modules/ROOT/pages/cache-scope.adoc

```diff
@@ -340,27 +368,86 @@ Mule provides the `InvalidatableCachingStrategy` interface, which allows you to
+====
+
+=== Examples: Invalidating Cache Usage
+
+==== Invalidate a Complete Cache
+
+To invalidate a cache, create a flow with two components:
+
+* An `http:listener` component that is the entry point of the flow.
+* An `ee:invalidate-cache` component that invalidates the referenced cache.
+
+Once this flow is executed, it invalidates the referenced cache by clearing all its stored entries.
+
+* XML Configuration file:
++
+[source,xml]
+----
+<flow name="invalidateCache">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-cache cachingStrategy-ref="InvalidatableCachingStrategy" />
+</flow>
+----
+
+==== Invalidate a Cache Key
+To invalidate a key, create a flow with two components:
+
+* An `http:listener` component that is the entry point of the flow.
+* An `ee:invalidate-key` component that calculates a cache key from the current event, and invalidates that key.
+
+Once this flow is executed, it calculates a key from the current event using the specified `keyGenerator-ref` or `keyGenerationExpression`, and invalidates that key.
+
+* XML Configuration file (`keyGenerator-ref` defined):
++
+[source,xml]
+----
+<spring:beans>
+       <spring:bean id="keyGenerator"
+                    class="com.mulesoft.mule.cache.config.CachingStrategyConfigTestCase.TestMuleEventKeyGenerator"/>
+</spring:beans>
+
+<flow name="invalidateKey">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-key cachingStrategy-ref="InvalidatableCachingStrategy" keyGenerator-ref="keyGenerator"/>
+</flow>
+----
+
+* XML Configuration file (`keyGenerationExpression` defined):
++
+[source,xml]
+----
+<flow name="invalidateKey">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/invalidate" doc:name="HTTP"/>
+        <ee:invalidate-key cachingStrategy-ref="InvalidatableCachingStrategy" keyGenerationExpression="#[payload]"/>
+</flow>
+----
+
+== Example: Cache Scope Usage
 
 The example that follows demonstrates the power of the cache scope with a Fibonacci function. The Fibonacci sequence is a series of numbers in which the next number in the series is always the sum of the two numbers preceding it.
 
 In this example, the Mule flow receives and performs two tasks for each request:
 
-. Executes, and returns the answer to, the Fibonacci equation (see below) using a number (_n_) provided by the caller as a request header. The images below show it as a query parameter, but it should be passed as a request header. Use a tool like https://www.getpostman.com/[Postman] to pass it the following equation text:  +
+. Executes, and returns the answer to, the Fibonacci equation (see below) using a number (_n_) provided by the caller as a request header. The images below show it as a query parameter, but it should be passed as a request header. Use a tool like Postman to pass it the following equation text:  +
```

Comment:
"Use a tool like Postman to pass it the following equation text" 
You could remove "it" in this case and leave it as _Use a tool like Postman to pass the following equation text_